docs/rms_oneliners.rst

```diff
@@ -0,0 +1,40 @@
+RMS "one-liner" functions
+=========================
+
+For lowerering the user threshold, some "one-liner" functions have been made for RMS. The purpose
+is both to make it simpler for users to export certain items, and in addition secure a better
+consistency. Hence the end user is not burdened to provide details, and only a script with quite
+a few lines will be needed.
+
+Exporting volumetrics from RMS
+------------------------------
+
+Volumetrics in RMS is always done by so-called volume jobs. The intention with the simplification
+is to use the RMS API behind the scene to retrieve all necessary data need for data io.
+
+Example:
+
+.. code-block:: python
+
+    from fmu.dataio.tools.rms import export_volumetrics
+    ...
+
+    # here 'Geogrid' is the grid model, and 'geogrid_volumes' is tha name of the volume job
+    outfiles = export_volumetrics(project, "Geogrid", "geogrid_volumes", global_config=CFG)
```

Comment:
still require `global_variables.yml` in a parsed state (dict), so the script is not really a one-liner. Replace this with a reader, and make the argument the filepath instead (and default it)?

Reply:
Badly explained... Try again:

Suggest that input argument `global_config` is the _path_ to global_variables.yml instead of the actual content (or allow both??), so that

1. Users with default locations of `global_variables.yml` won't have to do anything, and
2. other users can simply put the path, not having to parse it first in the same script, and
3. those who really want to parse it themselves can do that and pass the `dict`

Abstracting this would make it easier to move away from the whole usage of `global_variables.yml` as briefly discussed a few times.

Reply:
Yes, good idea